package.json

```diff
@@ -52,6 +52,7 @@
     "@storybook/addon-essentials": "7.6.10",
     "@storybook/addon-interactions": "^7.5.3",
     "@storybook/addon-storysource": "7.6.10",
+    "@storybook/addon-themes": "7.6.17",
```

Comment:
all sb versions needs to be on same version - please bump other packages as well